docs/cli.mdx

```diff
@@ -0,0 +1,204 @@
+
+To install the Svix CLI on Linux without a package manager:
+
+1. Download the latest Linux tar.gz file from <https://github.com/svix/svix-cli/releases/latest>
+
+2. Extract the file: `tar -xvf svix_X.X.X_Linux_x86_64.tar.gz`
+
+3. Run the executable: `./svix`
+
+</TabItem>
+<TabItem value="windows">
+
+To install the Svix CLI on Windows without Scoop:
+
+1. Download the latest Windows tar.gz file from <https://github.com/svix/svix-cli/releases/latest>
+
+2. Extract the `svix_X.X.X_Windows_x86_64.tar.gz` file.
+
+3. Run the `.exe` file
+
+</TabItem>
+</CLITabs>
+
+### 2) Login to your Svix Account
+
+After installing the Svix CLI, you must pair it with your Svix account.
+
+To do so, run `svix login` in the terminal. You will be prompted for your Svix Auth Token.
+
+Once supplied, this will be stored in your configuration file.
+
+:::info
+On most platforms, your config file can be found at `~/.config/svix/config.toml`.
+
+On Windows, your config file can be found at `%APPDATA%/svix/config.toml`
+:::
+
+If you would prefer not to store your AUTH_TOKEN on disk you can temporarily set it via the `SVIX_AUTH_TOKEN` env variable:
+
+ - On macOS and Linux: `export SVIX_AUTH_TOKEN=<MY-AUTH-TOKEN>`
+ - On Windows via command prompt: `set SVIX_AUTH_TOKEN=<MY-AUTH-TOKEN>`
+ - On Windows via PowerShell: `$env:SVIX_AUTH_TOKEN = '<MY-AUTH-TOKEN>'`
+
+### 3) Run your first command
+
+You’re now ready to use the Svix CLI. To get started let's create a new application:
+
+```sh
+svix application create --data-name "My Application"
+```
+
+This command creates a new application on Svix and returns the raw application object as JSON.
+
+### 4) Upgrade the Svix CLI
+
+<CLITabs>
+<TabItem value="homebrew">
+
+To upgrade the Svix CLI with [Hombrew](https://homebrew.sh), run:
+
+
+
+```sh
+brew upgrade svix/svix/svix
+```
+</TabItem>
+<TabItem value="scoop">
+
+To upgrade the Svix CLI with [Scoop](https://scoop.sh/), run:
+
+
+```sh
+scoop update svix
+```
+
+</TabItem>
+<TabItem value="macos">
+
+To upgrade the Svix CLI on macOS without Homebrew:
+
+1. Download the latest Darwin `tar.gz` file from <https://github.com/svix/svix-cli/releases/latest>
+
+2. Extract the file: `tar -xvf svix_X.X.X_Darwin_x86_64.tar.gz`
+
+2. Optionally, install the binary in a location where you can execute it globally (e.g., /usr/local/bin).
+
+</TabItem>
+<TabItem value="linux">
+
+To upgrade the Svix CLI on Linux without a package manager:
+
+
+1. Download the latest Linux `tar.gz` file from <https://github.com/svix/svix-cli/releases/latest>
+
+2. Extract the file: `tar -xvf svix_X.X.X_Linux_x86_64.tar.gz`
+
+3. Run the executable: `./svix`
+
+</TabItem>
+<TabItem value="windows">
+
+To upgrade the Svix CLI on Windows without Scoop:
+
+1. Download the latest Windows `tar.gz` file from <https://github.com/svix/svix-cli/releases/latest>
+
+2. Extract the `svix_X.X.X_Windows_x86_64.tar.gz` file.
+
+3. Run the `.exe` file
+
+</TabItem>
+</CLITabs>
+
+### 5) Get creative
```

Comment:
This is the main section of the tutorial, it needs to have more beef and merged with the section about the first command.
Don't say "run the first command", but rather something like:

First we'll create an app
`example - create app`
Let's see the app is showing
`example - list apps`
Or we can get it specifically
`example - get app`

Send message
`example - send message`

Reply:
Restructured to roughly follow this story.